### PR TITLE
plot_srf_square & map to python3

### PR DESCRIPTION
**Pull request by ttgenius**
Tested, to run with srf_python3 branch of slurm_gm_workflow
qcore/qcore/gmt still in python2, added list(map(xxx)) to enable plot_srf running in python3 env

### CODE REVIEW

File: visualization/gmt/plot_srf_square.py [hunk 1]
```diff
@@ -248,7 +248,7 @@ def kminch_scale():
         avg = round(np.average(row_data[:, 2]), 1)
         mx = round(max(row_data[:, 2]), 1)
         if r == 2:
-            mn, avg, mx = map(int, [mn, avg, mx])
+            mn, avg, mx = list(map(int, [mn, avg, mx]))
```
Comment (jpa198) [inline visualization/gmt/plot_srf_square.py]: Take this out then good to go

Comment (viktor76525) [reply visualization/gmt/plot_srf_square.py]: I agree, just waiting on this.

**Comment by ttgenius** [comment]
Still waiting on tests, maui being slow

**Comment by viktor76525** [comment]
What are you testing?

Python 3.6.5 (default, Feb 27 2019, 09:52:04) 
[GCC 8.2.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> a,b=map(int, ["1", "3"])
>>> a
1
>>> b
3
>>>

File: visualization/gmt/plot_srf_square.py [hunk 2]
```diff
@@ -26,7 +26,7 @@
 args = parser.parse_args()
 
 if args.plane_order is not None:
-    args.plane_order = list(map(int, args.plane_order.split('-')))
+    args.plane_order = map(int, args.plane_order.split('-'))
```
Comment (jpa198) [inline visualization/gmt/plot_srf_square.py]: Should still be a list

**Pull request merged**